### ISSUE TITLE
Cannot find module 'expect' from 'extend-expect.js'

### ISSUE THREAD

**Issue by azdanov** [kentcdodds/react-testing-library#34]
This only happens when I include `import 'react-testing-library/extend-expect'`. Otherwise `react-testing-library` works well. What might be the cause of this problem?

I am using latest `create-react-app`.

Error:
```js
 FAIL  src/App.test.js
  ● Test suite failed to run

    Cannot find module 'expect' from 'extend-expect.js'

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:179:17)
      at Object.<anonymous> (node_modules/react-testing-library/dist/extend-expect.js:7:15)
```

Failing test:
```js
it('should have a title', () => {
  const { getByText } = render(<App />);
  const text = 'Welcome to React';
  expect(getByText(text)).toBeInTheDOM();
});
```

Passing test:
```js
it('should have a title', () => {
  const { getByText } = render(<App />);
  const text = 'Welcome to React';
  expect(getByText(text).innerHTML).toBe(text);
});
```

**Comment by kentcdodds** [comment]
Ah, I know what's up. We don't need to import `expect` at all. It's a global. I'll fix it now.

**Comment by kentcdodds** [comment]
A fix should be published in a few minutes.

**Comment by azdanov** [comment]
Thanks, works as expected now!

**Comment by interglobalmedia** [comment]
Hi Kent, love your library. Am also taking the course. `.toBeInTheDOM()` is still in the REAME.md and was a bit confusing when I went in to find out what I could use in a test. However, I only found out that it was not the thing to use when I received a warning after running `npm test`. Perhaps, however, it might be nice if `.toBeInTheDOM()` was replaced with `.toBeIntheDocument() `in the `README.md`? Thanks!
